src/lpc.jl

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
In DSP.jl, the convention is to wrap separate functionality in modules and `@reexport`.